### PR TITLE
run: small improvements for accounting data output

### PR DESCRIPTION
**Pull request by YHNdnzj**

### CODE REVIEW

File: src/run/run.c [hunk 1]
```diff
@@ -1822,23 +1822,36 @@ static int start_transient_service(sd_bus *bus) {
...
 
-                        if (c.memory_peak != UINT64_MAX)
-                                log_info("Memory peak: %s", FORMAT_BYTES(c.memory_peak));
+                        if (c.memory_peak != UINT64_MAX) {
+                                const char *swap;
 
-                        if (c.memory_swap_peak != UINT64_MAX)
-                                log_info("Memory swap peak: %s", FORMAT_BYTES(c.memory_swap_peak));
+                                if (c.memory_swap_peak != UINT64_MAX)
+                                        swap = strjoina(" (swap: %s)", FORMAT_BYTES(c.memory_swap_peak));
```
Comment (poettering) [inline src/run/run.c:1829]: hmm, i think this should still contain the word "peak"?

Comment (YHNdnzj) [reply src/run/run.c:1829]: It feels redundant to me. And I used the same pattern in systemctl too (https://github.com/systemd/systemd/pull/30399)

**Comment by YHNdnzj** [comment]
The CI failure is #30910, unrelated.

**Review by bluca — APPROVED** [review]

File: src/run/run.c (hunk 1 above)
Comment (bluca) [reply src/run/run.c:1829]: yeah I think it makes sense to keep the output format in sync with systemctl, it's less confusing that way

**Pull request merged**